front/migrations/db/migration_112.sql

```diff
@@ -0,0 +1,7 @@
+-- Migration created on Nov 14, 2024
```

Comment:
you can wrap this with the `front/migration_with_backfill_template.sql`, to prevent applying it without running the backfill script first.

Reply:
I've seen this one used before, but I don't fully understand it. The few times I used it the think was complaining that I had to run a backfill first. 

In this case it doesn't feel super useful anyway as the migration will just naturally fail if the backfill hasn't been done first